### PR TITLE
Try to solve Issue #5483 pr #5627

### PR DESCRIPTION
**Pull request by PhilCattivocaratere**
I read PR #5627 and related #5483 Issue and try to solve. In my test Filter box appears and it is localized too.
Please put your attention to ToDoExplorerViewModel.cs where I had to sustitute
`... || CurrentUICulture.CompareInfo.IndexOf(...`
with
`... || CultureInfo.CurrentUICulture.CompareInfo.IndexOf(...`
to compile.

### CODE REVIEW

**Comment by retailcoder** [comment]
@IvenBach your comments are welcome here, I've added you as a requested reviewer. Thanks!

File: Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs [hunk 1]
```diff
@@ -305,5 +305,27 @@ public void Dispose()
                 _state.StateChanged -= HandleStateChanged;
             }
         }
+        private string _toDoFilter;
```
Comment (IvenBach) [inline Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs:308]: Minor item. Add a new line between the closing brace and `private string _toDoFilter`.

**Comment by IvenBach** [comment]
For consistency I'd say include the funnel icon as is done for [TestExplorer](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml#L208) and [InspectionResults](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/UI/Inspections/InspectionResultsControl.xaml#L152).

File: Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs (hunk 1 above)
Comment (retailcoder) [reply Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs:308]:
```suggestion

        private string _toDoFilter;
```

**Review by retailcoder — APPROVED** [review]

**Comment by PhilCattivocaratere** [comment]
I have missed funnel png image.

**Pull request merged**